apps/src/tutorialExplorer/tutorialSet.jsx

```diff
@@ -24,8 +24,30 @@ export default class TutorialSet extends React.Component {
   tutorialClicked = item =>
     this.setState({showingDetail: true, chosenItem: item});
 
-  tutorialDetailClosed = () =>
+  tutorialDetailClosed = () => {
+    // okay, so while the tutorialDetail window is open, it's possible that the focus has changed
+    // the user could've hit left/right arrows while the detail was open and moved focus, or also could've
+    // changed focus into the dialog, so before we wipe out the chosenItem, we find the associated
+    // item in the tutorial list and focus it, and also make sure to scroll it into view because it's possible
+    // the user navigated through enough of them to be off screen.
+    const tutorialDiv = document.querySelectorAll(
+      `[data-tutorial-code="${this.state.chosenItem.code}"]`
+    )[0];
+    if (
+      document.activeElement.hasAttribute('data-tutorial-code)') &&
+      document.activeElement !== tutorialDiv
+    ) {
+      // when in doubt, set timeout! _something_ is happening here that's fiddling with the focus, so we
+      // just defer it to the next run through of the even loop. if you remove this, the focus call will
+      // fail, although the scrollIntoView will work.
```

Comment:
NIT: regarding "if you remove this, the focus call will fail..."
It's a small thing, but using the "personal you" in this warning comment has the effect of implicitly assigning blame, which can make the next developer slightly less likely to trust themselves to change the code and therefore the code slightly less likely to be maintained - things we generally want to avoid. Using the "[generic you](https://ell.stackexchange.com/questions/110889/is-using-you-to-refer-to-anyone-not-the-person-youre-talking-to-a-known-sp)" has less of this effect and 3rd person has even less, so those may work better in this case. For example, rewriting this as "If setTimeout is removed, the the focus call will fail..." removes the implicit blame while retaining the warning.